featuretools/computational_backends/calculate_feature_matrix.py

```diff
@@ -762,6 +763,10 @@ def parallel_calculate_chunks(
     progress_callback=None,
     include_cutoff_time=True,
 ):
+    import_or_raise(
+        "dask",
```

Comment:
I think distributed would be a better check since it requires dask and is needed for this code section

Reply:
Updated